OrangeHRMTests/PageObjects/BasePage.cs

```diff
@@ -0,0 +1,58 @@
+﻿using OpenQA.Selenium;
+using OrangeHRMTests.Common.WebElements;
+using OrangeHRMTests.PageObjects.Modules;
+
+namespace OrangeHRMTests.PageObjects
+{
+    public class BasePage
+    {
+        public LeftMenuNavigationPanel LeftMenuNavigationPanel => new LeftMenuNavigationPanel();
+
+        private static MyWebElement AddButton = new MyWebElement(By.XPath("//*[contains(@class, 'orangehrm-header-container')]//button"));
```

Comment:
Just to confirm - do all these buttons are common for all pages that are inherited from BasePage?

Reply:
Yes